tests/test_run_ccoverage.py

```diff
@@ -0,0 +1,23 @@
+# coding=utf-8
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Test the run_ccoverage.py file."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals  # isort:skip
+
+import logging
+
+import funfuzz
+
+FUNFUZZ_LOG = logging.getLogger("run_ccoverage_test")
+logging.basicConfig(level=logging.DEBUG)
+logging.getLogger("flake8").setLevel(logging.WARNING)
```

Comment:
I think these settings will replace logging settings in other test files, since they are evaluated at import time, not run time. I'm not sure how to do it otherwise though.

Reply:
These seem to work for now, we can tweak more whenever we have a need to better optimise logging.